Sources/_StringProcessing/ByteCodeGen.swift

```diff
@@ -31,6 +35,9 @@ extension Compiler.ByteCodeGen {
     case let .backreference(ref):
       try emitBackreference(ref)
 
+    case let .symbolicReference(id):
+      emitUnresolvedReference(id: id)
```

Comment:
I wonder if the builder should be tracking this and resolve it, like it does for labels, captures, etc.

Reply:
Good call. Done.